packages/guides-code/resources/template/html/body/code.html.twig

```diff
@@ -0,0 +1,13 @@
+{% if node.raw %}
+    {# see the RawDirective for where this is coming from; a refactor is desired to move this onto its own template / renderer #}
```

Comment:
Wondering if this is still an issue. That would clean up this template.

Reply:
Not sure, that was already present before I entered the project.